mkdocs/README.md

```diff
@@ -0,0 +1,44 @@
+# Documentation
+
+The documentation is available at: [sbi-dev.github.io/sbi](http://sbi-dev.github.io/sbi)
+
+## Building the Documentation
+
+We use [`mike`](https://github.com/jimporter/mike) to manage, build, and deploy our
+documentation with [`mkdocs`](https://www.mkdocs.org/). To build the documentation
+locally, follow these steps:
+
+1. Install the documentation dependencies:
+
+    ```bash
+    python -m pip install .[doc]
+    ```
+
+2. Convert the current version of the documentation notebooks to markdown and build the
+   website locally using `mkdocs`:
+
+    ```bash
+    jupyter nbconvert --to markdown ../tutorials/*.ipynb --output-dir docs/tutorials/
+    mkdocs serve
+    ```
+
+### Deployment
+
```

Comment:
I see, this is just a backup for mkdocs. you can ignore the comment on the very top.